xla/service/gpu/amdgpu_compiler.cc

```diff
@@ -231,9 +231,10 @@ absl::Status AMDGPUCompiler::AddConvAndGemmAutotuningPasses(
     HloPassPipeline* pipeline, const se::GpuComputeCapability& gpu_version,
     const CompileOptions& options, HloModule* hlo_module,
     AutotuneConfig& autotune_config, tsl::thread::ThreadPool* thread_pool) {
-  if (GpuConvAlgorithmPicker::IsEnabled(hlo_module)) {
-    pipeline->AddPass<GpuConvAlgorithmPicker>(autotune_config);
-  }
+  //We do autotuning irrespective of --xla_gpu_autotune_level.
```

Comment:
Just to be clear, gemm will respect autotune level and seems conv autotune doesn't?